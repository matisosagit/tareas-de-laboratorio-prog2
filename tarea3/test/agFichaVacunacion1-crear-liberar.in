crearAGFichaVacunacion
liberarAGFichaVacunacion
crearAGFichaVacunacion
liberarAGFichaVacunacion
Fin