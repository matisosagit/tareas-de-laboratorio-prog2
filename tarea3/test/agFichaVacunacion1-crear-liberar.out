1> Se ejecut贸 'crearTAGFichaVacunacion' exitosamente.
2> Se ejecut贸 'liberarTAGFichaVacunacion' exitosamente.
3> Se ejecut贸 'crearTAGFichaVacunacion' exitosamente.
4> Se ejecut贸 'liberarTAGFichaVacunacion' exitosamente.
5> Fin.
